example/src/main/kotlin/com.expedia.graphql.sample/directives/DirectiveWiringFactory.kt

```diff
@@ -0,0 +1,22 @@
+package com.expedia.graphql.sample.directives
+
+import graphql.schema.idl.SchemaDirectiveWiring
+import graphql.schema.idl.SchemaDirectiveWiringEnvironment
+import graphql.schema.idl.WiringFactory
+
+class DirectiveWiringFactory : WiringFactory {
+    private val wiring = listOf(
+            StringEvalDirectiveWiring(),
+            CakeOnlyDirectiveWiring()
+    )
+
+    override fun providesSchemaDirectiveWiring(environment: SchemaDirectiveWiringEnvironment<*>): Boolean {
```

Comment:
lets modify those methods to expression body syntax (https://arturbosch.github.io/detekt/style.html#expressionbodysyntax) -> i.e. drop the `return` in favor of just `=`